language/move-lang/src/parser/syntax.rs

```diff
@@ -1141,6 +1143,24 @@ fn parse_type_parameter<'input>(tokens: &mut Lexer<'input>) -> Result<(Name, Kin
     Ok((n, kind))
 }
 
+// Parse optional type parameter list.
+//    TypeParameters = "<" Comma<TypeParameter> ">" | <empty>
```

Comment:
That should be "OptionalTypeParameters = ...."

Reply:
Done.